src/zeek-setup.cc

```diff
@@ -629,15 +629,22 @@ SetupResult setup(int argc, char** argv, Options* zopts)
 		supervisor_mgr = new Supervisor(std::move(cfg), std::move(*stem));
 		}
 
+	std::string seed_string;
+	if ( auto* seed_env = getenv("ZEEK_SEED") )
+		seed_string = seed_env;
+
 	const char* seed_load_file = getenv("ZEEK_SEED_FILE");
 
+	if ( seed_load_file && *seed_load_file && ! seed_string.empty())
```

Comment:
Does this need to take `-G` into account?

Reply:
Yep, I can see that - I thought `-G` would just take precedence, but I like the conflict better.